rich/console.py

```diff
@@ -522,14 +522,19 @@ def _is_jupyter() -> bool:  # pragma: no cover
         return False  # Other type (?)
 
 
-COLOR_SYSTEMS = {
+ConsoleColorSystem = Literal["auto", "standard", "256", "truecolor", "windows"]
+
+
+COLOR_SYSTEMS: Dict[ConsoleColorSystem, ColorSystem] = {
```

Comment:
just adding more specific typing to these 2 dicts while I'm there - we can do this now that the literal type has its own existence :slightly_smiling_face: